test/test_endpoints.py

```diff
@@ -352,3 +392,39 @@ def test_get_transaction_trace_of_rejected():
     resp_body = resp.json()
     assert resp_body["code"] == str(StarknetErrorCode.NO_TRACE)
     assert resp.status_code == 500
+
+
+@devnet_in_background()
```

Comment:
These tests expecting failure can probably be run without devnet_in_background, just relying on `app.test_client()` (see tests in this file for reference)

Reply:
I had trouble with that let's discuss it on a call

Reply:
Fixed in multiply commits just check test_endpoints.py file again